codecombat/codecombat
# Educator Resources impact page

## Features

 - A new CodeCombat static page featuring our fantastic partner stories on `/impact`
 - A new link in the nav bar for this "Impact" page.

## Tradeoffs

The page could have been componentized better but due to the short timeline it works well enough within the singular file, as many styles are pulled from our base-flat.sass file and bootstrap.

The page design is also not scoped for mobile, and thus only minimal work has been done to ensure the page is legible on a small screen width.

Finally there is a possibility of a user to be shown the "Request a Quote" in the nav bar. In this situation there are slightly too many links and there is an ugly state in the navbar shown below:

<img width="1051" alt="Screen Shot 2019-08-01 at 10 14 04 AM" src="https://user-images.githubusercontent.com/15080861/62313243-23ed2d00-b445-11e9-9663-ef083205f526.png">

We've chosen to punt fixing this as it doesn't impact users on the screen sizes we target.

We are also punting any manual analytics tracking for now.

## How to test

1. Check in this PR and run both `npm run dev` and `npm run proxy`.
2. Navigate to localhost:3000/impact or click on the Impact link from the home page.

## TODO:

 - [x] Before merging waiting for 3 more partner stories.
